packages/common/src/database/DbConnection.ts

```diff
@@ -76,6 +78,7 @@ export class DbConnection {
      */
     async one(repositoryData: RepositoryData, query: string) {
         query = addRepositorySchema(query, repositoryData)
+        console.log("query: ", query)
```

Comment:
Should we log this one differently?

Reply:
yes we should.  I am currently revising the logging and make it more flexible. No console logs anymore and adding log levels, and multiple outputs (I am using console and file output, as the file make it easier to find and collapse messages.
Will leave it for now, next PR is will change.